Source/SwiftLintFramework/Extensions/Configuration+LintableFiles.swift

```diff
@@ -13,14 +23,40 @@ extension Configuration {
             return [path]
         }
         let pathsForPath = included.isEmpty ? fileManager.filesToLint(inPath: path, rootDirectory: nil) : []
-        let excludedPaths = excluded.flatMap {
-            fileManager.filesToLint(inPath: $0, rootDirectory: rootPath)
-        }
+        let excludedPaths = excluded
+            .flatMap { resolveGlobs(in: $0) }
+            .flatMap {
+                fileManager.filesToLint(inPath: $0, rootDirectory: rootPath)
+            }
         let includedPaths = included.flatMap {
             fileManager.filesToLint(inPath: $0, rootDirectory: rootPath)
         }
         return (pathsForPath + includedPaths).filter {
             !excludedPaths.contains($0)
         }
     }
+
+    private func resolveGlobs(in pattern: String) -> [String] {
```

Comment:
Do you think it'd be worth it to extract this to a new class and add some tests directly?

Reply:
Done

Reply:
I left the first test since it's about Configuration calling glob, and added some new glob ones